### PR TITLE
Change default API Url

### PR DESCRIPTION
**Pull request by daviddemeij**
It took me some time to figure out that I should change the API URL to https://scihub.copernicus.eu/dhus. Maybe I'm misunderstanding this, but it seems to me that this should be the default URL.

### CODE REVIEW

**Comment by j08lue** [comment]
This is a misunderstanding. The right API URL to use for automated access is the one we have implemented, `/apihub`.

The reason why that one does not work for you and `/dhus` does is that for new user accounts or those that changed passwords, there is a period of a couple of weeks or so where that change has not yet propagated to `/apihub`. This is the weirdest thing and I don't know why this is happening.

The bottom line is: Use `/dhus` until your account works with `/apihub`. I will write to SciHub support to finally find out why they have this stupid issue.

Side note: `/dhus` seems to be faster when querying sub-products (e.g. files somewhere deeper inside the .SAFE zip archive).
